### ISSUE TITLE
Uncaught error when dangerouslySetInnerHTML attribute is removed

### ISSUE THREAD

**Issue by steelbrain** [developit/preact#90]
in `preact/src/dom/index.js`

```js
59	else if (name==='dangerouslySetInnerHTML') {
60		node.innerHTML = value.__html;
61	}
```

An uncaught error occurs there. When we have `dangerouslySetInnerHTML` attribute and we remove it, `preact/src/vdom/diff.js` calls `setAccessor` to reset the values. Because the value is undefined or null there, the error is thrown

**Comment by steelbrain** [comment]
Here is the full stack
```
TypeError: Cannot read property '__html' of null
    at setAccessor (/Users/steel/.atom/packages/linter-ui-default/node_modules/preact/dist/preact.js:437:27)
    at diffAttributes (/Users/steel/.atom/packages/linter-ui-default/node_modules/preact/dist/preact.js:797:6)
    at innerDiffNode (/Users/steel/.atom/packages/linter-ui-default/node_modules/preact/dist/preact.js:742:4)
    at diff (/Users/steel/.atom/packages/linter-ui-default/node_modules/preact/dist/preact.js:642:4)
    at innerDiffNode (/Users/steel/.atom/packages/linter-ui-default/node_modules/preact/dist/preact.js:712:14)
    at diff (/Users/steel/.atom/packages/linter-ui-default/node_modules/preact/dist/preact.js:642:4)
    at innerDiffNode (/Users/steel/.atom/packages/linter-ui-default/node_modules/preact/dist/preact.js:712:14)
    at diff (/Users/steel/.atom/packages/linter-ui-default/node_modules/preact/dist/preact.js:642:4)
    at renderComponent (/Users/steel/.atom/packages/linter-ui-default/node_modules/preact/dist/preact.js:965:14)
    at createComponentFromVNode (/Users/steel/.atom/packages/linter-ui-default/node_modules/preact/dist/preact.js:1045:4)
    at buildComponentFromVNode (/Users/steel/.atom/packages/linter-ui-default/node_modules/preact/dist/preact.js:1025:11)
    at diff (/Users/steel/.atom/packages/linter-ui-default/node_modules/preact/dist/preact.js:608:12)
    at innerDiffNode (/Users/steel/.atom/packages/linter-ui-default/node_modules/preact/dist/preact.js:712:14)
    at diff (/Users/steel/.atom/packages/linter-ui-default/node_modules/preact/dist/preact.js:642:4)
    at renderComponent (/Users/steel/.atom/packages/linter-ui-default/node_modules/preact/dist/preact.js:965:14)
    at MessagePort.rerender (/Users/steel/.atom/packages/linter-ui-default/node_modules/preact/dist/preact.js:361:19)%
```

@developit Can we have a quick release with the fix for this please?

**Comment by developit** [comment]
Released in [4.3.2](https://github.com/developit/preact/releases/tag/4.3.2)

**Comment by steelbrain** [comment]
Awesome, Thanks!
